services/background-worker/src/Wire/BackgroundWorker.hs

```diff
@@ -1,11 +1,17 @@
+{-# LANGUAGE BlockArguments #-}
+
 module Wire.BackgroundWorker where
 
+import Data.Domain
+import qualified Data.Map.Strict as Map
 import qualified Data.Metrics.Servant as Metrics
 import Imports
+import qualified Network.AMQP as Q
 import Network.Wai.Utilities.Server
 import Servant
 import Servant.Server.Generic
-import UnliftIO.Async
+import qualified System.Logger as Log
+import qualified System.Logger as Log'
```

Comment:
```suggestion
import qualified System.Logger as Log
```

Reply:
This will break the cleanup code logging. I need Log' so that I can pass in the logger to the IO code, rather than relying on the classy version.